README.md

```diff
@@ -15,11 +21,10 @@ The xiSEARCH algorithm is described in detail in [Mendez, Fischer *et al.* Mol.
 Scoring happens in three stages: 
 
 1. alpha candidates are selected and scored
-2. top n alpha candidates are taken and all matching beta-candidates (according to the precursormass) will be selected and prescored as pairs
+2. top n alpha candidates are taken and all matching beta-candidates (according to the precursor mass) will be selected and prescored as pairs
 3. the top X of these are then fully matched and scored
 
-The scoring function is applied to explain each spectrum with without considering if a peptide is target or decoy. In result for a false positive match the chance to to be target-target, target-decoy or a decoy decoy match should be 1:2:1.  Error control by false discovery rate estimation is then performed in a separate step with xiFDR.
-
+The scoring function is applied to explain each spectrum with without considering if a peptide is target or decoy. In result for a false positive match the chance to to be target-target, target-decoy or a decoy-decoy match should be 1:2:1.  Error control by false discovery rate estimation is then performed in a separate step with xiFDR.
```

Comment:
```suggestion
The scoring function is applied to explain each spectrum without considering if a peptide is target or decoy. In result for a false positive match the chance to to be target-target, target-decoy or a decoy-decoy match should be 1:2:1.  Error control by false discovery rate estimation is then performed in a separate step with xiFDR.
```

Reply:
"In result for a false positive match" this is not english. I think it should be "The resulting chances of each match to be"...

Reply:
updated